File: Data Structures & Algorithms/products-of-array-discluding-self/submission-1.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        int n=nums.length;
        int[] res=new int[n];
        for(int i=0;i<n;i++){
            int prd=1;
            for(int j=0;j<n;j++){
                if(i==j){
                    continue;
                }
                else{
                    prd=prd*nums[j];
                }
            }
            res[i]=prd;
        }
        return res;

    }
}  

File: Data Structures & Algorithms/longest-substring-without-duplicates/submission-1.py
class Solution:
    def lengthOfLongestSubstring(self, s: str) -> int:
        max_len = 0

        for i in range(len(s)):
            res = []

            for j in range(i, len(s)):
                if s[j] in res:
                    break
                res.append(s[j])

            max_len = max(max_len, len(res))

        return max_len